packages/common/src/node/UrlConverter.ts

```diff
@@ -182,6 +182,9 @@ class UrlConverter {
             // symbol 4 uses the last 6 bits of the third byte
             const numIndex = (bitCount / this.symbolNBits) | 0
             const num = nums[numIndex]
+            if (num === undefined) {
+                throw new Error(`Could not find number at index ${numIndex} of '${nums}'`)
```

Comment:
These should be `ProsopoEnvError('SOME.TRANSLATION_CODE', undefined, undefined, {numIndex, nums})` but we don't have a way of dropping variables into the translation currently so I'm fine for this change to be part of a follow piece on errors.

Reply:
yeah this is further justification that the error handling needs an overhaul imo. I'll add an issue to the gh board and ref this comment